Sources/ArcGISToolkit/Components/Popups/FieldsPopupElementView.swift

```diff
@@ -73,7 +73,8 @@ struct FieldsPopupElementView: View {
         
         var body: some View {
             if formattedValue.lowercased().starts(with: "http") {
-                Text(.init("[View](\(formattedValue))"))
+                Link("View", destination: URL(string: formattedValue)!)
```

Comment:
Instead of checking the start of the string and then force casting, perhaps it would be better to conditionally unwrap the created URL:

```swift
if let url = URL(string: formattedValue) {
    Link("View", destination: url)
} else {
    ...
}
```

Reply:
Or perhaps both the prefix and the conditional unwrap, in case the URL initializer might treat some non-URL strings as URLs.

Reply:
> in case the URL initializer might treat some non-URL strings as URLs.

Yup, that is what happens. So I'll follow the second suggestion.